### PR TITLE
fix: update int test readme to include adv test info

### PR DESCRIPTION
**Pull request by ray7yu**
Issue #, if available:
[GALI-999](https://sim.amazon.com/issues/GALI-999)

Description of changes:
Added information to integration test Readme regarding advanced integration tests and required prerequisite test resources.

Checklist:

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: main/integration-tests/README.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 ## Intro
 
-The integration-tests package is for running API tests against all SWB APIs. 
-These tests can run against local and dev environments during development. 
+The integration-tests package is for running API tests against all SWB APIs.
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md]: Can we say " The integration-tests package is **used** for running API tests against all SWB APIs."

File: main/integration-tests/README.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
...
+These tests can run against local and dev environments during development.
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md:6]: Is "during development" needed? I think it is understood.

File: main/integration-tests/README.md [hunk 3]
```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
...
+- **Test Administrator:** Create an internal admin-role user for running integration tests. (**Note the username and password**)
 
-* **Test Project:** Create a default project for running integration tests. (**Note the projectId**)
+- **Test Project:** Create a default project for running integration tests. (**Note the projectId**)
 
-* **AWS Budget:** Create a budget for the AWS account associated with the Test Project.
+- **AWS Budget:** Create a budget for the AWS account associated with the Test Project.
+
+#### Resources for Advanced Tests
+
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md:25]: Suggested:

- **Environment Type:** Import each specific Service Catalog product as an environment type.

- **Environment Configuration:** Create a configuration for every corresponding environment type that was created.

- **Data Source Study:** If testing BYOB studies, create an external BYOB study on the deployment with read and write access.

File: main/integration-tests/README.md [hunk 4]
```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
...
+- **Environment Type:** Import each specific Service Catalog Product as an Environment Type
+
+- **Environment Configuration:** Create a configuration for every corresponding environment type that was created
+
+- **Data Source Study:** If testing BYOB studies, create an external BYOB study on the deployment with Read & Write Access
 
 #### Resources to create through AWS Console:
 
-* **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)
+- **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md:34]: **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Speciify the parameter name. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)

"Use 'SecureString' for parameter type 'text' for data type." - This line is not clear.

File: main/integration-tests/README.md [hunk 5]
```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
...
 
-* **Cost Explorer:** Turn on Cost Explorer service, activate tags 'Env', 'Proj' anc 'createdBy'.
+- **Cost Explorer:** Turn on Cost Explorer service, activate tags 'Env', 'Proj' anc 'createdBy'.
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md]: Suggested: Enable Cost Explorer and activate the 'Env', 'Proj' anc 'createdBy' tags.

File: main/integration-tests/README.md [hunk 6]
```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
...
 
 ### Config File
 
 Once test resources are created, create a config file `main/integration-tests/config/settings/<STAGE>.yml` follow the example `main/integration-tests/config/settings/example.yml`.
 Use the same `<STAGE>` name as the main config file in `/main/config/settings`.
 
 Use Test Admin username, projectId, and Parameter Store name noted from the [Test Resources](#test-resources) section for field username, passwordPath and projectId in the config file.
-Use the same config values used in `/main/config/settings` for fields awsRegion, awsProfile, solutionName, envName, ane envType. 
+Use the same config values used in `/main/config/settings` for fields awsRegion, awsProfile, solutionName, envName, ane envType.
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md]: 1. should it be "and" at the end of sentence?
2. Use the same "configuration" values for the awsRegion, awsProfile, solutionName, envName, and envType fields?

File: main/integration-tests/README.md [hunk 7]
```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
...
+
+For each `{workspaceType}EnvTypeId` and `{workspacetype}ConfigId`, use the ids for the specific Environment Type and Environment Configuration created as Test Resources.
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md]: For each `{workspaceType}EnvTypeId` and `{workspacetype}ConfigId`, use the IDs for specific environment type and environment configuration created as test resources.

File: main/integration-tests/README.md [hunk 8]
```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
...
+
+For `byobStudy`, provide the id for the external Data Source study.
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md]: suggested: specify the ID for the external data source study.

File: main/integration-tests/README.md [hunk 9]
```diff
@@ -45,22 +58,26 @@ Once test resources and config file are created, you can run the integration tes
...
 
 ### Run against dev environment
+
 - In config file `main/integration-tests/config/settings/<STAGE>.yml`
   - set `isBuildServer` to `false`
   - set `isLocal` to `false`
- 
-##### run all integration tests from the root directory with: 
+
+##### run all integration tests from the root directory with:
```
Comment (shyogesh-sw) [inline main/integration-tests/README.md]: Capitalize "R" in run

**Review by SanketD92 — APPROVED** [review]
LGTM

**Pull request merged**